src/pages/Trainee/TraineeList.jsx

```diff
@@ -138,6 +139,50 @@ class TraineeList extends React.Component {
       });
     };
 
+    componentDidMount = () => {
+      const { data: { subscribeToMore } } = this.props;
+      subscribeToMore({
+        document: TRAINEE_UPDATED,
+        updateQuery: (prev, { subscriptionData }) => {
+          if (!subscriptionData) return prev;
+          const { getAllTrainee: { records } } = prev;
+          const { data: { traineeUpdated } } = subscriptionData;
+          const updateRecords = [...records].map((record) => {
+            if (record.originalId === traineeUpdated.originalId) {
+              return {
+                ...record,
+                ...traineeUpdated,
+              };
+            }
+            return record;
+          });
+          return {
+            getAllTrainee: {
+              ...prev.getAllTrainee,
+              count: prev.getAllTrainee.count - 1,
+              records: updateRecords,
+            },
+          };
+        },
+      });
+
+      subscribeToMore({
+        document: TRAINEE_DELETED,
+        updateQuery: (prev, { subscriptionData }) => {
+          if (!subscriptionData) return prev;
+          const { getAllTrainee: { records } } = prev;
```

Comment:
const { getAllTrainee: { records } } = prev;
Can be written as
const { getAllTrainee: { records }={} } = prev||{};